### DISCUSSION TITLE
Error trying to run Trivy in JeyBrains IDE

### DISCUSSION THREAD

**Discussion by digital-team-princestrust** [aquasecurity/trivy#6767] [Bugs]
### Description

Getting the following error when running Trivy in Jetbrains.  It runs and first shows the error below in red, then further results.
Any ideas on settings or is this a known bug, thanks? 

### Desired Behavior

Display results of findings, not return a error. 

### Actual Behavior

IDE Failed to deserialize the results file. No content to map due to end-of-input at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1]

Trivy Run completed, updating results.

/usr/local/bin/trivy fs --security-checks=config,vuln,secret --severity=CRITICAL,HIGH --offline-scan --ignore-unfixed --format=json --output=/tmp/Trivy6911009619090525634.json /home/me/public_html/projectfolder

### Reproduction Steps

```bash
1.Install Trivy in Jetbrains phpStorm
2.Go to Trivy findings in Menu
3.Click Run Trivy play button
4.view results in pop up
```

### Target

Filesystem

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
IDE Failed to deserialize the results file. No content to map due to end-of-input at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1]
```

### Operating System

Ubuntu 20.4

### Version

```bash
Version: 0.18.3
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by simar7** [comment]
This discussion board is for Trivy OSS. If your issue lies in the Trivy IntelliJ plugin you could open it [here](https://github.com/aquasecurity/intellij-trivy) but we are currently no longer maintaining it.
